addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -145,6 +145,31 @@ $bronze: #eea91e;
     .forum_answer img, .question-block img {
         max-width: 100%
     }
+
+    .o_wforum_index_entry_tags {
+        @for $size from 1 through length($o-colors) {
```

Comment:
- "$size" for an index?
- I am bit scared of this, what is the purpose of these new 50 rules with hardcoded colors ?

Reply:
I just used what was already used by others modules for coloring tags.
The color widget for a field shows specifics colors, and their ids is used in the class creation to get the right color.

Reply:
For the backend maybe, it is the website here. But ok, keep them and we'll see but adapt the rules correctly (see other comments)